### PR TITLE
xml: T6423: enforce priority on nodes having an owner

### PR DESCRIPTION
**Pull request by natali-rs1985**
## Change Summary

Added checking for priority tag if owner is defined in xml schema
If no priority then error is raised
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes)
- [ ] Migration from an old Vyatta component to vyos-1x, please link to related PR inside obsoleted component
- [x] Other (please describe): Added priority tag

## Related Task(s)

* https://vyos.dev/6423
* ## Related PR(s)

## Checklist:

- [x] I have read the [**CONTRIBUTING**](https://github.com/vyos/vyos-1x/blob/current/CONTRIBUTING.md) document
- [x] I have linked this PR to one or more Phabricator Task(s)
- [ ] I have run the components [**SMOKETESTS**](https://github.com/vyos/vyos-1x/tree/current/smoketest/scripts/cli) if applicable
- [x] My commit headlines contain a valid Task id
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly

### CODE REVIEW

File: interface-definitions/service_config-sync.xml.in [hunk 1]
```diff
@@ -5,6 +5,7 @@
       <node name="config-sync" owner="${vyos_conf_scripts_dir}/service_config-sync.py">
         <properties>
           <help>Configuration synchronization</help>
+          <priority>1</priority>
```
Comment (sever-sever) [inline interface-definitions/service_config-sync.xml.in]: use please 1000+ priority here
It should be triggered after all other nodes are configured
It checks if something is changed in a configured node section/subsection and then applies this if changed.

Comment (natali-rs1985) [reply interface-definitions/service_config-sync.xml.in]: Done

**Comment by jestabro** [comment]
This is not critical, but with these changes, we can remove the misplaced priority elements _outside_ of owner blocks:
```
From 5f952848fab64b9e24db355ea15ef15c0977aa85 Mon Sep 17 00:00:00 2001
From: John Estabrook <jestabro@vyos.io>
Date: Thu, 6 Jun 2024 10:14:32 -0500
Subject: [PATCH] remove orphan priority elements

---
 interface-definitions/load-balancing_wan.xml.in          | 1 -
 interface-definitions/service_aws_glb.xml.in             | 1 -
 interface-definitions/service_monitoring_telegraf.xml.in | 1 -
 3 files changed, 3 deletions(-)

diff --git a/interface-definitions/load-balancing_wan.xml.in b/interface-definitions/load-balancing_wan.xml.in
index 485899d26..310aa0343 100644
--- a/interface-definitions/load-balancing_wan.xml.in
+++ b/interface-definitions/load-balancing_wan.xml.in
@@ -3,7 +3,6 @@
   <node name="load-balancing">
     <properties>
       <help>Configure load-balancing</help>
-      <priority>900</priority>
     </properties>
     <children>
       <node name="wan" owner="${vyos_conf_scripts_dir}/load-balancing_wan.py">
diff --git a/interface-definitions/service_aws_glb.xml.in b/interface-definitions/service_aws_glb.xml.in
index 4541bee1a..71de1f03a 100644
--- a/interface-definitions/service_aws_glb.xml.in
+++ b/interface-definitions/service_aws_glb.xml.in
@@ -5,7 +5,6 @@
       <node name="aws">
         <properties>
           <help>Amazon Web Service</help>
-          <priority>1280</priority>
         </properties>
         <children>
           <node name="glb" owner="${vyos_conf_scripts_dir}/service_aws_glb.py">
diff --git a/interface-definitions/service_monitoring_telegraf.xml.in b/interface-definitions/service_monitoring_telegraf.xml.in
index 7d9e5525c..2624023ea 100644
--- a/interface-definitions/service_monitoring_telegraf.xml.in
+++ b/interface-definitions/service_monitoring_telegraf.xml.in
@@ -5,7 +5,6 @@
       <node name="monitoring">
         <properties>
           <help>Monitoring services</help>
-          <priority>1280</priority>
         </properties>
         <children>
           <node name="telegraf" owner="${vyos_conf_scripts_dir}/service_monitoring_telegraf.py">
-- 
2.44.0
```

**Review by dmbaturin — APPROVED** [review]

**Review by jestabro — APPROVED** [review]

**Comment by c-po** [comment]
@MergifyIo backport sagitta

**Pull request merged**